src/state/lists/hooks.ts

```diff
@@ -71,8 +71,13 @@ export function listToTokenMap(list: TokenList): TokenAddressMap {
             return { ...list.tags[tagId], id: tagId }
           })
           ?.filter((x): x is TagInfo => Boolean(x)) ?? []
+
       const token = new WrappedTokenInfo(tokenInfo, tags)
-      if (tokenMap[token.chainId][token.address] !== undefined) throw Error('Duplicate tokens.')
+      if (tokenMap[token.chainId][token.address] !== undefined) {
+        console.warn(`Duplicate token skipped: ${token.address}`)
```

Comment:
Is this warning useful ?

Reply:
If using throw Error, when has duplicate tokens, in our FE will keep calling and website will get stuck.